Makefile

```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 
 include build/makelib/common.mk
-include build/makelib/helm.mk
```

Comment:
Does it need to be included on line 100, or should we keep it here at the top of the file for readability?

Reply:
I prefer the top, so I'll change that.